frontend/src/components/grids/AssetGrid.tsx

```diff
@@ -1,18 +1,31 @@
 import React, { ReactElement } from "react";
-import { AssetT } from "../../types";
+import Imgix from "react-imgix";
+import { AssetT, SourceT } from "../../types";
 import "../../styles/Grid.css";
 interface Props {
   assets: AssetT[];
 }
 
 export function AssetGrid({ assets }: Props): ReactElement {
   const gridItems = assets.map((asset, idx) => {
+    console.log(asset);
+
     return (
-      <div className="ix-grid-item" key={`${asset.filename}-${idx}`}>
+      <div className="ix-grid-item" key={`${asset.id}-${idx}`}>
         <div className="ix-grid-item-image">
-          <img src={asset.url} alt={asset.filename} />
+          <Imgix
+            src={asset.attributes.origin_path}
+            width={340}
+            height={340}
+            imgixParams={{
+              auto: "format",
+              fit: "crop",
+              crop: "entropy",
```

Comment:
Maybe this is better?
```suggestion
              crop: "faces,entropy",
```